src/main/java/com/devotedmc/ExilePearl/ExilePearlApi.java

```diff
@@ -95,12 +95,6 @@
 	 */
 	boolean isCitadelEnabled();
 
-	/**
```

Comment:
Unless we're abandoning non Artemis / Zeus support, why not leave CivChat Enabled in there? Just have add to the implementation of API the isArtemisEnabled() check, and if that's enabled, civchat definitely is not, etc.

Reply:
With this commit https://github.com/DevotedMC/ExilePearl/commit/6a02adaf37a4aa83eee2a19efbd3ceb51fbf854b we pretty much ripped out anything CivChat2, this commit had some leftovers

Reply:
oooh, ok. Hadn't reviewed more broadly, that is totally my bad. Ignore this one.